### PR TITLE
AssimpImporter: Simplify skinMerge() test

### PR DESCRIPTION
**Pull request by Squareys**
**This PR is against `next`!**

Hi @mosra & @pezcode !

Thank you both for the amazing work on #99, here's a small contribution with the hand-crafted gltf file replacing the leaky scene copy.

Best,
Jonathan

### CODE REVIEW

File: src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp [hunk 1]
```diff
@@ -1426,47 +1426,10 @@ void AssimpImporterTest::skinMergeEmpty() {
...
-    CORRADE_VERIFY(original);
-    aiScene* scene{};
-
-    /* aiCopyScene() is for some reason not present in builds without export
-       functionality (which we have on Windows CIs). The code inside does this
-       and then sets some private mIsCopy field which we don't care about.
-       Unfortunately, <assimp/SceneCombiner.h> is available only since 4.0, so
-       on the ancient 3.2 use the C API (and assume nobody needs the insane corner case of running the test against Assimp 3.2 that has export APIs
-       disabled). */
-    #ifdef ASSIMP_HAS_SCENE_COMBINER
```
Comment (mosra) [inline src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp:1448]: The `#include` block related to this should get cleaned up as well. I'll do it during merge.

Otherwise looks great, thanks a lot! :+1:

**Comment by mosra** [comment]
I went ahead and rewrote also the Assimp-specific matrix operations and squashed the whole thing into 9bf2b2b0b1151903b21170490fe7dc55e91dc529. Thanks a lot!
